include/dts-environment.sh

```diff
@@ -4,16 +4,32 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-# Text Reset
-COLOR_OFF='\033[0m'
+# Text colors:
+NORMAL='\033[0m'
 RED='\033[0;31m'
 YELLOW='\033[0;33m'
 GREEN='\033[0;32m'
-
-BOARD_VENDOR="$(dmidecode -s system-manufacturer)"
+BLUE='\033[0;36m'
+
+# DTS options:
+HCL_REPORT_OPT="1"
+DASHARO_FIRM_OPT="2"
+REST_FIRM_OPT="3"
+DES_KEYS_OPT="4"
+REBOOT_OPT_UP="R"
+REBOOT_OPT_LOW="r"
```

Comment:
If we have to define upper and lower, we can also convert one to another: https://stackoverflow.com/questions/2264428/how-to-convert-a-string-to-lower-case-in-bash